src/main/java/org/apache/accumulo/testing/performance/tests/ScanExecutorPT.java

```diff
@@ -178,11 +185,10 @@ private long scan(String tableName, AccumuloClient c, AtomicBoolean stop,
     return count;
   }
 
-  private LongSummaryStatistics runShortScans(Environment env, String tableName, int numScans)
-      throws InterruptedException, ExecutionException {
+  private LongSummaryStatistics runShortScans(Environment env, String tableName, int numScans) {
 
-    Map<String,String> execHints = ImmutableMap.of("executor", "se2");
-    Map<String,String> prioHints = ImmutableMap.of("priority", "1");
+    Map<String,String> execHints = ImmutableMap.of("scan_type", "se2p1");
+    Map<String,String> prioHints = ImmutableMap.of("scan_type", "se1p1");
```

Comment:
Could change the map names and use more descriptive name for the scan types.

```suggestion
    Map<String,String> dHints = ImmutableMap.of("scan_type", "dedicated");
    Map<String,String> sHints = ImmutableMap.of("scan_type", "short");
```